app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.post = @post
+    @comment.author = current_user
+
+    if @comment.save
+      redirect_to "/users/#{current_user.id}/comments", notice: 'Comment was successfully created.'
+    else
+      render 'posts/show'
+    end
+  end
```

Comment:
- [ ] It would be nice if you make a new method in your controller and route as suggested by the previous reviewer to make your application follow the requirement which was mentioned that you should have forms for creating a comment. I know you have a comment form but it should be on the new route to follow professionalism and the actual way of making forms :blush: